include/grpcpp/opencensus.h

```diff
@@ -19,6 +19,8 @@
 #ifndef GRPCPP_OPENCENSUS_H
 #define GRPCPP_OPENCENSUS_H
 
+#include "opencensus/trace/span.h"
+
```

Comment:
Because of issues including the Abseil dependence, let me recommend that there should be an explicit check that we're actually building with bazel. Something like:

``` cpp
#ifndef GRPC_BAZEL_BUILD
#error OpenCensus for gRPC is only supported when building with bazel
#endif
```

Reply:
Done.